### ISSUE TITLE
Tilda crash after hiding

### ISSUE THREAD

**Issue by marcgenou** [lanoxx/tilda#431]
New MATE 1.24.1 updates and tilda now crashes after hiding

Tilda has started. Press F12 to pull down the window.

(tilda:6388): tilda-ERROR **: 09:29:17.364: X Error: BadMatch (invalid parameter attributes)
`trap' para punto de parada/seguimiento (`core' generado)

Any idea about this one?

**Comment by kabel-voidship** [comment]
I can confirm this:
```
(tilda:33326): tilda-ERROR **: 22:04:12.801: X Error: BadMatch (invalid parameter attributes)
Trace/breakpoint trap (core dumped)

pflergleheimer@boomzilla:~/.config/tilda$ tilda --version
Tilda 1.5.2


```
1.24.1

Any other useful information, please let me know.

**Comment by kabel-voidship** [comment]
Core dump:
```

                Stack trace of thread 40249:
                #0  0x00007f55ffe5a314 g_log_writer_default (libglib-2.0.so.0 + 0x5b314)
                #1  0x00007f55ffe57a29 g_log_structured_array (libglib-2.0.so.0 + 0x58a29)
                #2  0x00007f55ffe57f81 g_log_structured_standard (libglib-2.0.so.0 + 0x58f81)
                #3  0x000055f9eb5a907a n/a (tilda + 0x1a07a)
                #4  0x00007f55ffc7882b _XError (libX11.so.6 + 0x4382b)
                #5  0x00007f55ffc755d8 n/a (libX11.so.6 + 0x405d8)
                #6  0x00007f55ffc75675 n/a (libX11.so.6 + 0x40675)
                #7  0x00007f55ffc75fda _XEventsQueued (libX11.so.6 + 0x40fda)
                #8  0x00007f55ffc67992 XPending (libX11.so.6 + 0x32992)
                #9  0x00007f560023e0f3 n/a (libgdk-3.so.0 + 0x8d0f3)
                #10 0x00007f55ffe4edda g_main_context_check (libglib-2.0.so.0 + 0x4fdda)
                #11 0x00007f55ffe9f10b n/a (libglib-2.0.so.0 + 0xa010b)
                #12 0x00007f55ffe4fc03 g_main_loop_run (libglib-2.0.so.0 + 0x50c03)
                #13 0x00007f560048f21f gtk_main (libgtk-3.so.0 + 0x1e021f)
                #14 0x000055f9eb5997c9 n/a (tilda + 0xa7c9)
                #15 0x00007f55ffa72152 __libc_start_main (libc.so.6 + 0x28152)
                #16 0x000055f9eb599e6e n/a (tilda + 0xae6e)
                
                Stack trace of thread 40255:
                #0  0x00007f55ffb3f46f __poll (libc.so.6 + 0xf546f)
                #1  0x00007f55ffe9f168 n/a (libglib-2.0.so.0 + 0xa0168)
                #2  0x00007f55ffe50221 g_main_context_iteration (libglib-2.0.so.0 + 0x51221)
                #3  0x00007f55ffe50272 n/a (libglib-2.0.so.0 + 0x51272)
                #4  0x00007f55ffe79511 n/a (libglib-2.0.so.0 + 0x7a511)
                #5  0x00007f55ffc1c3e9 start_thread (libpthread.so.0 + 0x93e9)
                #6  0x00007f55ffb4a293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 40296:
                #0  0x00007f55ffb44d5d syscall (libc.so.6 + 0xfad5d)
                #1  0x00007f55ffe99c9b g_cond_wait_until (libglib-2.0.so.0 + 0x9ac9b)
                #2  0x00007f55ffe21803 n/a (libglib-2.0.so.0 + 0x22803)
                #3  0x00007f55ffe21994 g_async_queue_timeout_pop (libglib-2.0.so.0 + 0x22994)
                #4  0x00007f55ffe7dcaa n/a (libglib-2.0.so.0 + 0x7ecaa)
                #5  0x00007f55ffe79511 n/a (libglib-2.0.so.0 + 0x7a511)
                #6  0x00007f55ffc1c3e9 start_thread (libpthread.so.0 + 0x93e9)
                #7  0x00007f55ffb4a293 __clone (libc.so.6 + 0x100293)
                
                Stack trace of thread 40256:
                #0  0x00007f55ffb3f46f __poll (libc.so.6 + 0xf546f)
                #1  0x00007f55ffe9f168 n/a (libglib-2.0.so.0 + 0xa0168)
                #2  0x00007f55ffe4fc03 g_main_loop_run (libglib-2.0.so.0 + 0x50c03)
                #3  0x00007f56000821a8 n/a (libgio-2.0.so.0 + 0x1001a8)
                #4  0x00007f55ffe79511 n/a (libglib-2.0.so.0 + 0x7a511)
                #5  0x00007f55ffc1c3e9 start_thread (libpthread.so.0 + 0x93e9)
                #6  0x00007f55ffb4a293 __clone (libc.so.6 + 0x100293)

```

**Comment by ensky0** [comment]
I use Debian testing, MATE 1.24.1.
I can confirm this, too.

```
ensky0@home:~/.config$ tilda
Tilda has started. Press F1 to pull down the window.

(tilda:10237): tilda-ERROR **: 08:37:28.355: X Error: BadMatch (invalid parameter attributes)
Trace/breakpoint trap
ensky0@home:~/.config$ tilda --version
Tilda 1.5.2

Copyright (c) 2012-2020 Sebastian Geiger (lanoxx@gmx.net)
Copyright (c) 2005-2009 Tristan Sloughter (sloutri@iit.edu)
Copyright (c) 2005-2009 Ira W. Snyder (tilda@irasnyder.com)

General Information: https://github.com/lanoxx/tilda
Bug Reports: https://github.com/lanoxx/tilda/issues?state=open

This program comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it
under certain conditions. See the file COPYING for details.
```

**Comment by holyorb2** [comment]
Linux 5.8.11-1-MANJARO
` tilda-ERROR **: 16:34:07.047: X Error: BadMatch (invalid parameter attributes)
Trace/breakpoint trap (core dumped)
`

**Comment by ranjan-purbey** [comment]
Possible duplicate of https://github.com/lanoxx/tilda/issues/428

**Comment by ranjan-purbey** [comment]
Disabling "Always on Top" option apparently fixes the crash for me

**Comment by lanoxx** [comment]
This issue should be fixed in version 1.5.3 through commit f7d5ffd0381cb3815873efcf15d53ae0c7e16a98.
